client/flutter/lib/l10n/intl_en_US.arb

```diff
@@ -3,16 +3,31 @@
   "commonWorldHealthOrganization": "World Health Organization",
   "commonWorldHealthOrganizationCoronavirusApp": "COVID-19 App",
   "commonWhoAppShareIconButtonDescription": "Check out the official COVID-19 Guide App https://whocoronavirus.org/app",
-  "commonWorldHealthOrganizationCoronavirusAppVersion": "Version 0.1 (12412)\n© 2020 WHO",
+  "commonWorldHealthOrganizationCoronavirusAppVersion": "Version {version} ({buildNumber})",
+  "commonWorldHealthOrganizationCoronavirusCopyright": "© {year} WHO",
   "commonPermissionRequestPageButtonSkip": "Skip",
+  "commonDialogButtonOk": "OK",
+  "commonContentLoadingDialogUpdateRequiredTitle": "App Update Required",
+  "commonContentLoadingDialogUpdateRequiredBodyText": "Please update to the latest version of the app in order to receive the latest information and updates.",
+  "@section": "Legal Landing Page",
+  "legalLandingPageTitle": "Official WHO COVID-19 Information App",
+  "legalLandingPageButtonGetStarted": "Get Started",
+  "legalLandingPageButtonAgree": "By proceeding, you agree to our",
```

Comment:
must preserve the newline
```suggestion
  "legalLandingPageButtonAgree": "By proceeding, you agree to our\n",
```

Reply:
same as above, I can put it back but the more strings like that we have the more difficult it'll be to localize the app.